assets/libraries/openapi/library.rego

```diff
@@ -52,25 +52,37 @@ is_ref(schema) {
 	object.get(schema, "$ref", "undefined") != "undefined"
 }
 
-check_string_schema(doc, s, type, field) {
+check_schema(doc, s, type, field) {
 	component_schema := doc.components.schemas[sc]
 	sc == s
 	component_schema.properties[p].type == type
 	object.get(component_schema.properties[p], field, "undefined") == "undefined"
 }
 
-undefined_properties_in_schema(doc, schema, type, field) {
+undefined_properties_in_schema(doc, value, sc_kind, type, field) {
```

Comment:
@rafaela-soares can you take this opportunity to document these auxiliary functions we have?

Reply:
Yes, of course. Sorry